src/types/strings/printable.rs

```diff
@@ -15,21 +20,25 @@ impl StaticPermittedAlphabet for PrintableString {
         b'8', b'9', b' ', b'\'', b'(', b')', b'+', b',', b'-', b'.', b'/', b':', b'=', b'?',
     ]);
 
-    fn chars(&self) -> Box<dyn Iterator<Item = u32> + '_> {
-        Box::from(self.0.iter().map(|byte| *byte as u32))
-    }
-
     fn push_char(&mut self, ch: u32) {
         debug_assert!(
             Self::CHARACTER_SET.contains(&ch),
-            "{} not in character set",
-            ch
+            "{ch} not in character set"
         );
         self.0.push(ch as u8);
     }
+
+    fn chars(&self) -> Box<dyn Iterator<Item = u32> + '_> {
+        Box::from(self.0.iter().map(|byte| *byte as u32))
+    }
```

Comment:
Why `u32` when it can fit in a `u8`? Also why use `Box<dyn Iterator>` instead of `impl Iterator`?

Reply:
Those were already there, I just changed the order to be same as in Trait. I have no idea 😅 

https://github.com/XAMPPRocky/rasn/blob/a7a00c76d702e530a14d517d830adc899bd31dc4/src/types/strings/printable.rs#L18-L20

I can change those at the same time.

Reply:
Oh I didn't notice this was in the trait, in that case it makes sense because you can't use either of those.